jest.config.js

```diff
@@ -1,5 +1,19 @@
+const path = require('path');
+const fs = require('fs');
 const { createConfig } = require('@edx/frontend-build');
 
+/** in case there isn't an env.config.js in development or pushed into the master branch
+    we want to rely on a fallback config that exists in ./jest that provides an empty object.
+    This implementation was first done in frontend-build
```

Comment:
I'm confused by this line of the comment

Reply:
I went ahead and tried explaining it more clearly but the gist is: 
The new config logic in `jest.config.js` will search for an `env.config.js` in the root directory, and if it doesn't exist then it will use the filepath to a fallback `env.config.js` that exists in the `./jest` directory. The latter is an empty object because we assume that the codebase is intentionally lacking an `env.config.js` during development or a GitHub action, and the goal here is to just not upset Jest when we assign an `env.config` in the `moduleNameMapper` property below.

Reply:
Note that the conversation happening [in a comment below](https://github.com/openedx/frontend-app-learner-record/pull/275/files/9788e72583d194e7614b5efc755e0e85debf622b#r1480588270) will very likely lead to any changes in this file to be removed for the PR.